Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -1638,12 +1794,17 @@ def fetch_incidents(client, first_fetch_time, last_run: dict = None):
 
     for raw_incident in raw_incidents:
         incident_id = raw_incident.get('incident_id')
+        incident_data = get_incident_extra_data_command(client, {"incident_id": incident_id,
```

Comment:
lets add a new boolean parameter `fetch incident alerts and artifacts` with default value True, and if this param enabled, call get extra data.

Reply:
done